pkgs/development/python-modules/rapidfuzz/default.nix

```diff
@@ -51,6 +51,8 @@ buildPythonPackage rec {
 
   preBuild = ''
     export RAPIDFUZZ_BUILD_EXTENSION=1
+  '' + lib.optionalString (stdenv.isDarwin && stdenv.isx86_64) ''
+    export CMAKE_ARGS="-DCMAKE_CXX_COMPILER_AR=$AR -DCMAKE_CXX_COMPILER_RANLIB=$RANLIB"
   '';
```

Comment:
rapidfuzz `v2.13.3` will now always set these variables when they are not set yet: https://github.com/maxbachmann/RapidFuzz/commit/fe82774ae7388d27d499f6774aa8a8ba184f4134. So this should not be needed after upgrading it here in nixos

Reply:
Thank you for looking into this, and thank you for printing out those variables too for easier debugging. This is what I notice when I remove the overrides (formatted):

```
CMAKE_AR="/nix/store/xazz27ylsw2304sifcn3d8y2n9vjjy0w-clang-wrapper-11.1.0/bin/ar" ;
CMAKE_C_COMPILER_AR="CMAKE_C_COMPILER_AR-NOTFOUND" ;
CMAKE_CXX_COMPILER_AR="CMAKE_CXX_COMPILER_AR-NOTFOUND";                                                                             
CMAKE_RANLIB="/nix/store/xazz27ylsw2304sifcn3d8y2n9vjjy0w-clang-wrapper-11.1.0/bin/ranlib" ;
CMAKE_C_COMPILER_RANLIB="CMAKE_C_COMPILER_RANLIB-NOTFOUND" ;
CMAKE_CXX_COMPILER_RANLIB="CMAKE_CXX_COMPILER_RANLIB-NOTFOUND"
```

The build still fails, I assume because, for example, `CMAKE_CXX_COMPILER_AR` is not the empty string. I have no idea why this is happening right now myself.

Reply:
Yes I guess in this case setting it yourself is probably the right solution. Not sure where this is set right now.